### PR TITLE
Configure "ensure_fully_multiline" rule at `.php_cs.dist.twig`

### PR DESCRIPTION
**Pull request by phansys**
For a sample diff, check https://github.com/sonata-project/SonataAdminBundle/compare/3.x...phansys:method_argument_space?expand=1.

### CODE REVIEW

File: src/Console/Command/DispatchCommand.php [hunk 1]
```diff
@@ -373,7 +373,10 @@ private function dispatchFiles(Package $package): void
             }
             // If the previous branch is not merged into the current one, do nothing.
             if ($previousBranch && $this->githubClient->repos()->commits()->compare(
-                    static::GITHUB_GROUP, $repositoryName, $currentBranch, $previousBranch
+                static::GITHUB_GROUP,
+                $repositoryName,
+                $currentBranch,
+                $previousBranch
                 )['ahead_by']) {
```
Comment (greg0ire) [inline src/Console/Command/DispatchCommand.php]: There is an indention issue here.

Comment (OskarStark) [reply src/Console/Command/DispatchCommand.php]: If I am correct the indention is not checked by this rule automatically.

Anyway I like the result but we should check this after the dev kit updates and FlintCI fix PRs

Comment (greg0ire) [reply src/Console/Command/DispatchCommand.php]: Yes, you're correct IIRC, we will have to check. Luckily, people seldom violate that rule.

Comment (soullivaneuh) [reply src/Console/Command/DispatchCommand.php]: In any case, there is an indentation issue here. No? :-)

Comment (OskarStark) [reply src/Console/Command/DispatchCommand.php]: @phansys can you please fix this?

Comment (phansys) [reply src/Console/Command/DispatchCommand.php]: The proposed fix doesn't look right to me (https://flintci.io/repositories/50/analyses/24032):
```diff
@@ -373,11 +373,11 @@ final class DispatchCommand extends AbstractNeedApplyCommand
             }
             // If the previous branch is not merged into the current one, do nothing.
             if ($previousBranch && $this->githubClient->repos()->commits()->compare(
-                    static::GITHUB_GROUP,
-                    $repositoryName,
-                    $currentBranch,
-                    $previousBranch
-                )['ahead_by']
+                static::GITHUB_GROUP,
+                $repositoryName,
+                $currentBranch,
+                $previousBranch
+            )['ahead_by']
             ) {
                 continue;
             }
```
Maybe we should squash this call into a single line.

Comment (OskarStark) [reply src/Console/Command/DispatchCommand.php]: Can you post an issue to the PHP-CS-Fixer guys? Maybe it gets fixed by another release? 

@Soullivaneuh which version is FlintCI using?

**Comment by OskarStark** [comment]
Shall we proceed here and check the results afterwards in the DevKit PR’s or close entirely?

**Comment by greg0ire** [comment]
I would proceed… the cases will only be edge cases IMO.

**Comment by OskarStark** [comment]
Me too 👍🏻

**Pull request merged**